nixos/modules/profiles/hardened.nix

```diff
@@ -115,4 +114,8 @@ with lib;
   # Ignore outgoing ICMP redirects (this is ipv4 only)
   boot.kernel.sysctl."net.ipv4.conf.all.send_redirects" = mkDefault false;
   boot.kernel.sysctl."net.ipv4.conf.default.send_redirects" = mkDefault false;
+  # Disabling this so  an attacker cannot gain root access by passing init=/bin/sh as a kernel parameter.
+  boot.loader.systemd-boot.editor = false;
```

Comment:
This should use `mkDefault`

Reply:
(and is useless, this is not a compile time option.)

Reply:
> (and is useless, this is not a compile time option.)

What do you mean?

Reply:
`editor = false` is not a security feature, it is just cosmetic.